templates/commands/upgrade/flags.go

```diff
@@ -101,6 +107,11 @@ func (f *Flags) Register(set *cli.FlagSet) {
 		Target:  &f.ResumeFrom,
 		Usage:   "begin or resume the upgrade starting at this manifest file",
 	})
+	u.BoolVar(&cli.BoolVar{
+		Name:   "even-if-up-to-date",
```

Comment:
The name is a bit hard to read. Would something like `--force-upgrade` or `--force-refresh` better?

Reply:
`--continue-if-current` is :chefkiss: . Thanks! Changed.